### PR TITLE
Add a custom field to serialize choices properly

### PR DESCRIPTION
**Pull request by Kapin**
Fixes all the issues with storing choices in the database and retrieving
them

### CODE REVIEW

File: src-backend/api/generator.py [hunk 1]
```diff
@@ -56,10 +57,14 @@ def __get_properties(self):
             props['numeric'] = self.element.numeric
 
         if self.element.choices:
-            props['choices'] = self.element.choices
+            props['choices'] = self.__parse_choices()
 
         return props
 
+    def __parse_choices(self):
+        choice_list = json.loads(self.element.choices)
```
Comment (Trinovantes) [inline src-backend/api/generator.py]: so our database stores the choices as a json array?

File: src-backend/api/generator.py [hunk 2]
```diff
@@ -56,10 +57,14 @@ def __get_properties(self):
...
+        return ','.join(choice_list)
```
Comment (Trinovantes) [inline src-backend/api/generator.py]: what does this return (in terms of format)?

File: src-backend/api/generator.py (hunk 1 above)
Comment (Kapin) [inline src-backend/api/generator.py]: currently yes, this is probably going to change.

File: src-backend/api/generator.py (hunk 2 above)
Comment (Kapin) [inline src-backend/api/generator.py]: choice1,choice2,choice3

**Comment by Trinovantes** [comment]
+1

**Comment by ConnorCimowsky** [comment]
LGTM

**Pull request merged**